### PR TITLE
Translations from weblate

### PR DESCRIPTION
**Pull request by zetok**

### CODE REVIEW

File: translations/de.ts [hunk 1]
```diff
@@ -223,7 +231,7 @@ Zu hohe Auflösungen können daher zu Problemen in Videoanrufen führen.</transl
         <source>qTox needs to use the Tox DNS, but can&apos;t do it through a proxy.
     Ignore the proxy and connect to the Internet directly?</source>
         <translation>qTox verwendet Tox-DNS, um eine Verbindung herzustellen, kann das bei aktiviertem Proxyserver aber nicht.
-Soll der Proxyserver ignoriert werden und eine Direktverbindung hergestellt werden? </translation>
+   Soll der Proxyserver ignoriert werden und eine Direktverbindung hergestellt werden?</translation>
```
Comment (sudden6) [inline translations/de.ts:234]: Only whitespace changed makes no sense, doesn't it?

File: translations/de.ts [hunk 2]
```diff
@@ -322,7 +325,7 @@ Viele Grüße,
     <message>
         <source>Form</source>
         <translatorcomment>Ausgelassen</translatorcomment>
-        <translation></translation>
+        <translation>Von</translation>
```
Comment (sudden6) [inline translations/de.ts:328]: `Form` doesn't translate to `Von`, typo in source, meaning `From` ?

File: translations/de.ts [hunk 3]
```diff
@@ -332,12 +335,12 @@ Viele Grüße,
     <message>
         <source>Someone</source>
```
Comment (sudden6) [inline translations/de.ts:336]: The next four look like placeholders, do we really use them?

File: translations/de.ts [hunk 4]
```diff
@@ -583,7 +595,7 @@ Disabling chat history now will leave the encrypted history intact (but not usab
     <message>
         <source>Form</source>
         <translatorcomment>Ausgelassen</translatorcomment>
-        <translation></translation>
+        <translation>Von</translation>
```
Comment (sudden6) [inline translations/de.ts:598]: `Form` != `Von` again typo in source?

File: translations/de.ts [hunk 5]
```diff
@@ -754,7 +766,7 @@ Disabling chat history now will leave the encrypted history intact (but not usab
     <message>
         <source>Move to circle...</source>
         <comment>Menu to move a friend into a different circle</comment>
-        <translation>In Kreis verschieben</translation>
+        <translation>Verschieben in Kreis...</translation>
```
Comment (sudden6) [inline translations/de.ts:769]: I think the previous one was better.

File: tools/update-translation-files.sh [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+# Script for updating translation files.  Should be ran after
+# translatable strings are modified.
+#
+# Needed, since weblate cannot into doing that before translation.
```
Comment (sudden6) [inline tools/update-translation-files.sh]: `Needed, since weblate cannot do that before translation.` is better english I think.

Comment (zetok) [inline tools/update-translation-files.sh]: ~done

**Pull request merged**